android/settings.gradle

```diff
@@ -1,3 +1,5 @@
 rootProject.name = 'shop'
 apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
 include ':app'
+include ':react-native-vector-icons'
```

Comment:
You've followed the steps for an older version of React-native.
Please check the original document, how to properly link vector-icons in react-native > 0.60+

Reply:
in Android/app/build.gardle:

project.ext.vectoricons = [
    iconFontNames: [ 'MaterialIcons.ttf', 'Ionicons.ttf' ] 
]

apply from: "../../node_modules/react-native-vector-icons/fonts.gradle"

as given in latest documents

Reply:
But you've copied these things in **settings.gradle**, manually added fonts in assets directory, which is not required